config.py

```
@@ -0,0 +1,139 @@
+        0.6,
+        ge=0.0,
+        le=2.0,
+        description="Low temperature for late generations (exploitation)"
+    )
+
+    @property
+    def total_requests_needed(self) -> int:
+        """Calculate total LLM calls needed for one complete evolution run.
+
+        Returns:
+            Total number of API calls (population_size × num_generations)
```

Ruff (0.14.1)

Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)